### ISSUE TITLE
Playback stops after a while if app is on background or device is locked

### ISSUE THREAD

**Issue by MoisesValera** [apache/cordova-plugin-media#224]
On Oreo (Android 8.0+), playback stops after a while whenever app goes into background mode or device is locked, it's impossible to restore it by reloading app or unlocking device. 

I´ve read that this can be fixed by implementing a Foreground Service, how could it be done?, any guidance will be great.

**Comment by MoisesValera** [comment]
Found a solution by mixing this plugin with [a Cordova foreground service](https://github.com/DavidBriglio/cordova-plugin-foreground-service), OS killing app on background will stops for Android 8.0+

**Comment by fdambrosio** [comment]
I have the same problem, have you fixed it using Cordova foreground service?

**Comment by MoisesValera** [comment]
@fdambrosio Yes, completely but service ongoing notification is not optional, also avoid killing the service once you start it

**Comment by fdambrosio** [comment]
@MoisesValera thanks, but what do you think about using both cordova-music-controls-plugin notification (audio streaming app) and Cordova foreground service (mandatory) notification?

**Comment by MoisesValera** [comment]
@fdambrosio Well i know music controls plugin is really fancy but using both is not an alternative for Oreo, lets try building a version of music controls using only Foreground Services
